lib/block_view.go

```diff
@@ -2715,8 +2736,7 @@ func (bav *UtxoView) _disconnectAcceptNFTBid(
 
 	// (1) Set the old NFT entry.
 	if operationData.PrevNFTEntry == nil || operationData.PrevNFTEntry.isDeleted {
-		return fmt.Errorf("_disconnectAcceptNFTBid: prev NFT entry doesn't exist; " +
-			"this should never happen")
+		return fmt.Errorf("prev NFT entry doesn't exist; this should never happen")
```

Comment:
In core, errors are generally prefixed with the name of the function returning the error in order to make it easier to track them down (since you basically always have a pseudo-stack trace).  I would add "_helpDisconnectNFTSold: " to the error messages here for consistency.